bedrock/firefox/templates/firefox/firstrun/firstrun-quantum.html

```diff
@@ -4,23 +4,44 @@
 
 {% add_lang_files "firefox/new/quantum" %}
 
-{% extends "firefox/firstrun/base-pebbles.html" %}
+{% from "macros.html" import fxa_email_form, fxa_link_fragment with context %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
 
 {% block body_id %}firstrun-quantum-onboarding{% endblock %}
 
 {% block page_title %}
   {{ _('Welcome to Firefox') }}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {{ css_bundle('firefox_firstrun_quantum') }}
 {% endblock %}
 
+{% block gtm_page_id %}data-gtm-page-id="/firefox/firstrun/"{% endblock %}
+
+{% block extrahead %}
+  {% if switch('firefox-update-notification-modal', ['en-US']) %}
+    {{ css_bundle('firefox-update-notification-firstrun-whatsnew') }}
+  {% endif %}
+{% endblock %}
+
+{# Bug 1387984 #}
+{% block update_notification %}
+  {% if switch('firefox-update-notification-modal', ['en-US']) %}
+    {% include 'includes/firefox-update-instructions.html' %}
+    {{ js_bundle('firefox-update-notification-firstrun-whatsnew') }}
+  {% endif %}
+{% endblock %}
+
 {% block global_nav %}{% endblock %}
 {% block site_header %}{% endblock %}
 
 {% block content %}
-  <div id="scene">
+  <div id="scene" class="t-firefox">
```

Comment:
Is this still needed now all pages default to Firefox branding?

Reply:
Nope, and I ended up not even doing anything with this class anyway, this is an artifact. I'll remove.